plugins/obs-ffmpeg/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 project(obs-ffmpeg)
 
 option(ENABLE_FFMPEG_LOGGING "Enables obs-ffmpeg logging" OFF)
+option(USE_NEW_MPEGTS_OUTPUT "Use native srt/rist mpegts output" ON)
```

Comment:
I think this option should follow the same convention as others (`ENABLE_`) and be named `ENABLE_NEW_MPEGTS`.

Reply:
cmake overhaul had not been merged iirc.
But yeah, it's better to follow the pattern of other options.